capstone/src/main/webapp/reference.html

```diff
@@ -3,21 +3,80 @@
   <head>
     <title>Reference</title>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <noscript><link rel="stylesheet" href="assets/css/noscript.css"></noscript>
   </head>
-  <body>
-    <h2>References for UniViz</h2>
-    <p>
-      During the course of creating UniViz, the following resources were used:
-    </p>
-    <ul>
-      <li>Stellar HTML5/CSS3 template from <a href="html5up.net/stellar">html5up.net</a></li>
-      <li><a href="https://collegescorecard.ed.gov/data/documentation/">The CollegeScoreCard API</a></li>
-      <li>The following Google APIs:
-        <ul>
-          <li>Google Charts API</li>
-          <li>Google Places API</li>
-        </ul>
-      </li>
-    </ul>
+  <body class="is-preload">
+    <div id="wrapper">
+      <header id="header">
+        <h2>References for UniViz</h2>
+      </header>
+      <div id="main">
+        <section id="content" class="main">
+          <section>
+            <div>
+              <p>
+              During the course of creating UniViz, the following resources were used:
+              </p>
+              <ul>
+                <li>Stellar HTML5/CSS3 template from <a href="html5up.net/stellar">html5up.net</a></li>
+                <li><a href="https://collegescorecard.ed.gov/data/documentation/">The CollegeScoreCard API</a></li>
+                <li>The following Google APIs:
+                  <ul>
+                    <li>Google Charts API</li>
+                    <li>Google Places API</li>
```

Comment:
```suggestion
                    <li>Google Maps JavaScript API</li>
```
we ended up not using Places, so replace it with Google Maps JavaScript API